core/serviceContainer/serviceContainer_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"testing"
```

Comment:
We did not follow the "service container" pattern. Maybe remove the whole service container stuff